CONTRIBUTING.md

```diff
@@ -0,0 +1,69 @@
+# Contributing guide
+
+**Want to contribute? Great!** 
+We try to make it easy, and all contributions, even the smaller ones, are more than welcome.
+This includes bug reports, fixes, documentation, examples... 
+But first, read this page.
+
+## Legal
+
+All original contributions to Narayana are licensed under the
+[GNU LESSER GENERAL PUBLIC LICENSE Version 2.1](https://github.com/jbosstm/narayana/blob/master/LICENSE).
+
+All contributions are subject to the [Developer Certificate of Origin (DCO)](https://developercertificate.org/).
```

Comment:
I think this could be socialised on the Narayana forum for some further feedback first but I would be fine with it

Reply:
I raised the question (https://groups.google.com/g/narayana-users/c/rLR6w2RudIk). If there is still no input from the community by the end of the week then I will assume that's tacit agreement and we can go ahead with the mandate [of contributions being subject to the DCO].